webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -682,8 +682,13 @@ export const GlideTable: React.FC<GlideTableProps> = ({
               '',
             )}`;
             break;
+          case V1LocationType.TRAINING:
+            dataPath = `bestTrial.summaryMetrics.avgMetrics.${currentColumn.column.replace(
+              'training',
```

Comment:
fix: this needs to be `'training.'`, with a period at the end so that it parses correctly.